### DISCUSSION TITLE
How run custom program?

### DISCUSSION THREAD

**Discussion by POMXARK** [winapps-org/winapps#138] [Support]
How run Adobe XD ?
How to create an additional auto setting?

**Comment by KernelGhost** [comment]
You have two main options:
1. Run `winapps manual C:\Path\To\Adobe\XD.exe`
2. Add Adobe XD as a [supported application](https://github.com/winapps-org/winapps?tab=readme-ov-file#adding-additional-pre-defined-applications).
